plugins/module_utils/redfish_utils.py

```diff
@@ -2262,7 +2270,7 @@ def virtual_media_insert_via_patch(self, options, param_map, uri, data, image_on
             return response
         return {'ret': True, 'changed': True, 'msg': "VirtualMedia inserted"}
 
-    def virtual_media_insert(self, options):
+    def virtual_media_insert(self, resource_type, options):
```

Comment:
You must add new arguments at the end, otherwise it is impossible to keep backwards compatibility.
```suggestion
    def virtual_media_insert(self, options, resource_type='Manager'):
```

Reply:
Thank you for your suggestion, I will update this.